tasks/devcontainer.py

```diff
@@ -167,3 +170,39 @@ def is_up(ctx) -> bool:
 def is_installed(ctx) -> bool:
     res = ctx.run("which devcontainer", hide=True, warn=True)
     return res.ok
+
+
+def run_on_devcontainer(func):
+    """
+    This decorator will run the decorated function in a devcontainer if the selected platform is linux.
+    All you need to do is to decorate your task with this decorator and add a `platform` argument to your task.
+    """
+
+    @wraps(func)
+    def _run_on_devcontainer(ctx, *args, **kwargs):
+        if kwargs.get('platform'):
+            platform = kwargs['platform'].lower()
+            if platform == "linux" and py_platform.system().lower() != platform:
+                # If we choose to run them on linux, and we are not on linux already
+                if not file().exists():
```

Comment:
This is not related to this PR: what about a little more precise name instead of `file` (eg `devcontainer_file`). It could be slightly confusing

Reply:
I see your point but this `file` is already in the `devcontainer` module so that should be fine, users outside this module will use `devcontainer.file` and I think that's fine. wdyt?